spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -43,17 +47,22 @@
     end
   end
 
-  context 'email_analytics_attributes' do
-    before(:each) do
-      stub_request_passed_proofing_results
-    end
```

Comment:
this stub was causing only 'passed' emails to be sent, no matter what the test calling this shared example had configured